opsmngr/opsmngr.go

```diff
@@ -195,3 +199,30 @@ func (c *Client) Do(ctx context.Context, req *http.Request, v interface{}) (*atl
 	}
 	return response, err
 }
+
+func setListOptions(s string, opt interface{}) (string, error) {
```

Comment:
[nit] can we rename this to 
```suggestion
func setQueryParams(s string, opt interface{}) (string, error) {
```
it makes more sense now that we use it for more than lists